corehq/apps/userreports/app_manager/data_source_meta.py

```diff
@@ -103,9 +104,9 @@ def get_filter(self):
 
 class FormDataSourceMeta(AppDataSourceMeta):
 
-    def __init__(self, domain, data_source_type, data_source_id):
-        super(FormDataSourceMeta, self).__init__(domain, data_source_type, data_source_id)
-        self.source_form = Form.get_form(self.data_source_id)
+    def __init__(self, domain, app_id, data_source_type, data_source_id):
+        super().__init__(domain, app_id, data_source_type, data_source_id)
+        self.source_form = self.app.get_form(self.data_source_id)
```

Comment:
not at all related to your PR: Its really confusing that the reference to `data_source_id` in `corehq/apps/userreports` is not the ID of a UCR data source, but the id of a form or a case type. There's a comment about it in the parent class and I get the logic here, but still confusing

Reply:
That is probably why I initially thought this task would involve migrating data sources :-p

Not going to change in this PR, but agreed.